### DISCUSSION TITLE
WTF Solidity 教程推荐语收集

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#521] [General]
WTF Solidity 入门篇（前30讲）将由科学出版社出版，感谢WTF贡献者和读者！

现在我们需要收集本教程的推荐语，如果大家觉得WTF Solidity有帮助到你，就请花2分钟写一两句话，选中的推荐语会印在书里。

再次感谢大家的支持！我们会继续努力！

![WechatIMG3756](https://github.com/AmazingAng/WTF-Solidity/assets/14728591/abf78693-2c30-4653-a684-5b5e0b78990b)

**Comment by yanboishere** [comment]
wow 恭喜恭喜！

**Comment by yanboishere** [comment]
WTF Solidity 入门篇是Solidity学习者的首选教程！
作为GitHub上Stars数最多Solidity教程，本套教程以简洁明了的方式解释了Solidity的核心概念，并通过相当多的的示例帮助我从零开始编写智能合约，逐渐由Web2进入Web3世界。感谢所有贡献者对该教程的贡献！

**Comment by 0xKaso** [comment]
这本书使用简明易懂的语言，以多个案例的形式对solidity语法进行了解读，适合初学者学习。它还汇集了多位资深开发者的贡献，内容质量和措辞的严谨程度有所保障，是一本可靠的solidity教程参考书。如果您正在学习solidity或者想要深入理解智能合约开发，推荐大家阅读。

**Comment by Silence-dream** [comment]
这本书的独特之处在于它集结了100多名教程内容贡献者的智慧和经验。这意味着读者可以从不同的视角和实践经验中学习，获得更加全面和深入的知识。每一个教程都经过认真审查，确保内容准确、易于理解，并配有示例代码和解释说明。这使得读者可以通过实际动手编写代码来加深对Solidity语言和web3技术的理解。

**Comment by 0xkookoo** [comment]
这本书对于新手和老手来说都是一个宝贵的资源，可以帮助他们系统地学习智能合约。对于初学者，这本教程提供了一个循序渐进的学习路径，帮助他们从零开始理解智能合约的基本概念和原理。对于经验丰富的开发者，这本书可以作为一个查漏补缺的指南，帮助他们深入挖掘智能合约开发的高级主题。

**Comment by DestinyWei** [comment]
WTF Solidity 入门篇为读者打开了通往 Web3 世界的大门。通过该教程的学习，我们可以很轻松地理解 Solidity 的语法与使用，同时每个章节都有相对应的案例代码和解释说明帮助读者们更好地吸收与理解，是一本不可多得的 Solidity 入门教材与指南！

**Comment by junjieora** [comment]
它是一本书，一个朦胧的梦，得以让未来照进现实

**Comment by FireLoveInIce** [comment]
探索未来的编程之旅，尽在《WTF-Solidity》！这本引人入胜的书籍将带领您进入web3的神秘世界。无论您是从传统互联网转型还是寻求新的编程挑战，这本书都是您不可或缺的指南。

作者将枯燥的编程概念转化为令人兴奋的实例和引人入胜的案例。您将亲身体验如何创建智能合约、交互式DApp和无与伦比的去中心化体验。这本书将让您的代码在web3的世界中发挥最大的潜力。

无论您是初学者还是有经验的开发者，这本书都将激发您的创造力和探索精神。准备好迎接web3的挑战吗？《WTF-Solidity》将成为您最值得信赖的伙伴，引领您走向编程世界的未来！

**Comment by skyMetaverse** [comment]
无论你是初学者还是有经验的Solidity开发者，WTF-Solidity以其独特的教学风格，将Solidity的学习变得有趣而易于理解。
并且不仅教授了基础的Solidity知识，还教授最佳实践和常见陷阱，让你在项目实践中写出安全可靠的智能合约！

**Comment by zhiyuan2007** [comment]
本书是进入web3领域，接触defi，了解智能合约，学习solidity语言的最好入门参考书之一。内容全是干货，每个知识点都有对应的案例，学习起来不会枯燥。

**Comment by willisXu** [comment]
WTF Solidity或许华语世界中，最好的Web3编程入门课程！
虽简单易懂、类比活泼生动，但轻轻松松学习完后，已能掌握Solidity的完整基础功
助力Web2程序员、编程小白成功进入Web3编程世界的大门
实属Geek极客精神的程序钜作，符合Web3世界崇尚的开放、共享的社区氛围

**Comment by zhengqian8958** [comment]
千里之行，始于足下。而你的Web3之旅，将从这里开启

**Comment by hattiepwvi** [comment]
.

**Comment by evilcos** [comment]
我把 WTF Solidity 教程推给了许多朋友，尤其是准备从传统安全转入 Web3 安全的白帽黑客朋友们。通过 WTF Solidity 教程不仅我自己快速掌握了许多 Solidity 要点，我的白帽黑客朋友们也大受启发。感谢这种思维碰撞，很好很直接！

**Comment by Fankouzu** [comment]
当今区块链技术正在迅速发展，而Solidity是以太坊平台上最流行的智能合约编程语言之一。如果你想掌握Solidity语言的编程技能，那么《WTF Solidity 入门篇》是一本非常好的书籍选择。

这本书提供了一个全面的实用指南，涵盖了从Solidity编程基础到高级概念的所有内容。它不仅提供了丰富的实例和案例，而且还包括了大量的编程练习，帮助读者深入理解Solidity语言的核心概念。

《WTF Solidity 入门篇》油100多名内容贡献者共同汇集内容和知识经验，涵盖了智能合约作为区块链Dapp应用开发所需要的所有知识点，读者可以全面理解Solidity语言的特点和优势。

总之，如果你想成为一名优秀的Solidity开发者，那么《WTF Solidity 入门篇》是一本不可或缺的参考书籍。

**Comment by Frank2333333** [comment]
如果你想进入区块链世界，那么学习solidity是一条通往真实区块链世界的捷径，而《WTF Solidity》这本书正是帮你走上这条捷径的绝佳指南！它以简洁明了的方式介绍了 Solidity 的核心概念和语法，并结合实际示例帮助读者理解和应用这些概念。无论你是刚入门还是已有一定经验的区块链开发者，这本书都能够帮助你快速掌握 Solidity，并开始尝试构建自己的智能合约。

**Comment by power-cake** [comment]
牛逼克拉斯的web3教程来了

**Comment by 0xFooork** [comment]
恭喜载誉而归，实至名归
